### ISSUE TITLE
Delete records after 2 days

### ISSUE THREAD

**Issue by mjswart-d2l** [mjswart/CollectWaitStats#7]
1 day doesn't give the full story of a whole day, and 1 week seems like too much data

**Comment by fatherjack** [comment]
I'd say for a week profile then the 1min collection frequency is too quick. There seems to me, to be a difference between troubleshooting (fine detail over a short duration) and bench-marking (less frequent samples over a longer period).

**Comment by mjswart-d2l** [comment]
You're right... I think I want the defaults to match the week-long benchmark/profile use case.
